HackIllinois/ViewControllers/HIUserDetailViewController.swift

```diff
@@ -110,7 +110,8 @@ extension HIUserDetailViewController {
             let user = HIApplicationStateController.shared.user,
             let url = user.qrURL,
             !UserDefaults.standard.bool(forKey: HIConstants.PASS_PROMPTED_KEY(user: user)) else { return }
-        HIAPI.PassService.getPass(with: url.absoluteString)
+        let passUrl = "\(url.absoluteString)&identifier=\(user.email)"
+        HIAPI.PassService.getPass(with: passUrl)
```

Comment:
I'm assuming this new url works with the api

Reply:
The URL is used only for the pass service, the QR code stays the same. This is so we can display the user's email on the pass